source/API_Reference/SMTP_API/errors_and_troubleshooting.md

```diff
@@ -0,0 +1,136 @@
+---
+seo:
+  title: Errors and Troubleshooting
+  description: SMTP Response codes and troubleshooting tips
+  keywords: SMTP, send email, integrate, building, filters, scheduling, substitution, suppression groups, unique arguments, recipients
+title: Errors and Troubleshooting
+weight: 0
+layout: page
+navigation:
+  show: true
+---
+
+- [Response codes](#-Response-codes)
+- [Turning off click tracking](#-Turning-off-click-tracking)
+- [Invalid SMTP API Header](#-Invalid-SMTP-API-Hheader)
+- [Certificate verification failed for smtp.sendgrid.net](#-Certificate-verification-failed-for-smtp.sendgrid.net)
+
+{% anchor h3 %}
+Response codes
+{% endanchor %}
+
+Each SMTP call you make returns a response. `200` responses are usually success responses, `400` are usually deferrals, and we continue to retry `400` messages for up to 72 hours, and `500` responses are hard failures that are not retried by our servers. This table has possible response codes with example errors and a general explanation of that sort of response. This is not an all-inclusive list -  if you are getting an error you don't recognize, check out our [Deliverability center](https://sendgrid.com/deliverabilitycenter/#/).
```

Comment:
Each SMTP call you make returns a response. `200` responses are usually success responses, and `400` responses are usually deferrals. SendGrid continues to retry resending `400` messages for up to 72 hours. `500` responses are hard failures that are not retried by our servers.